src/v2/Apps/RSS/rssServerApp.tsx

```diff
@@ -0,0 +1,35 @@
+import express from "express"
+import { fetchQuery } from "relay-runtime"
+import { createRelaySSREnvironment } from "v2/System/Relay/createRelaySSREnvironment"
+import { RSS_PARTNER_UPDATES_QUERY } from "./queries/RssPartnerUpdatesQuery"
+import { RssPartnerUpdatesQuery } from "v2/__generated__/RssPartnerUpdatesQuery.graphql"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { getENV } from "v2/Utils/getENV"
+import { GALLERY_PARTNER_UPDATES_CHANNEL } from "config"
+
+const rssServerApp = express()
+
+rssServerApp.set("views", `${__dirname}/templates`)
+rssServerApp.set("view engine", "ejs")
+
+rssServerApp.get("/rss/partner-updates", async (req, res) => {
+  const relayEnvironment = createRelaySSREnvironment({
+    userAgent: req.header("User-Agent"),
+  })
+
+  const { articlesConnection } = await fetchQuery<RssPartnerUpdatesQuery>(
+    relayEnvironment,
+    RSS_PARTNER_UPDATES_QUERY,
+    { channelId: GALLERY_PARTNER_UPDATES_CHANNEL }
+  )
```

Comment:
I believe this is the first instance of us just using Relay to fetch server-side without React? So no need for that `metaphysics2` function.

Reply:
Could also just be a normal `fetch`, I suppose.

Reply:
generated relay types are always good